x-pack/plugins/infra/types/eui.d.ts

```diff
@@ -164,6 +164,7 @@ declare module '@elastic/eui' {
     disabled?: boolean;
     isLoading?: boolean;
     dateFormat?: string;
+    isCustom?: boolean;
```

Comment:
This change is in:

```
infra/types/eui.d.ts
```

Infra? I take it for this to work we had to add this `isCustom` to within infra since their plugins is controlling the types for `EuiDatePickerRangeProps`?

Reply:
Yes it is global for now